src/steps/oppsummeringssteg/Oppsummeringssteg.tsx

```diff
@@ -192,33 +186,162 @@ export default function Oppsummeringssteg({
                                 )}
                             />
                         </div>
-                        <div style={{ marginTop: '2rem' }}>
-                            <Oppsummeringsfelt
-                                feltNavn="Opphold på institusjon"
-                                feltVerdi={oppsummeringInstitusjon(
-                                    institusjonsopphold.borPåInstitusjon,
-                                    institusjonsopphold.periode
-                                )}
-                            />
-                        </div>
+                        <Oppsummeringsfelt
+                            feltNavn="Kvalifiseringsprogrammet"
+                            feltVerdi={oppsummeringDeltarIKvp(
+                                kvalifiseringsprogram.deltar,
+                                kvalifiseringsprogram.periode
+                            )}
+                        />
                     </Accordion.Content>
                 </Accordion.Item>
                 <Accordion.Item defaultOpen>
                     <Accordion.Header>Andre utbetalinger</Accordion.Header>
                     <Accordion.Content>
                         <Oppsummeringsfelt
-                            feltNavn="Pensjonsordninger"
-                            feltVerdi={oppsummeringPensjonsordninger(
-                                pensjonsordning.mottarEllerSøktPensjonsordning,
-                                pensjonsordning
-                            )}
+                            feltNavn="Andreutbetalinger"
+                            feltVerdi={visSvarTilAndreUtbetalinger(mottarAndreUtbetalinger)}
                         />
-                        <div style={{ marginTop: '2rem' }}>
-                            <Oppsummeringsfelt
-                                feltNavn="Etterlønn"
-                                feltVerdi={oppsummeringEtterlønn(etterlønn.mottarEllerSøktEtterlønn, etterlønn)}
+                            {
+                                sykepenger.periode ? (
+                                    <div style={{ marginTop: '2rem' }}>
+                                        <Oppsummeringsfelt
+                                            feltNavn="Sykepenger"
+                                            feltVerdi={`Ja, jeg mottar sykepenger i perioden ${formatPeriode(sykepenger.periode!)}`}
+                                        />
```

Comment:
Lag gjerne en funksjon for hver av feltverdiene, slik som tidligere var gjort for _oppsummeringEtterlønn()_. Det blir mer lesbart enn alle ternary-funksjonene.